ambry-messageformat/src/test/java/com.github.ambry.messageformat/MessageFormatInputStreamTest.java

```diff
@@ -59,9 +59,9 @@ private void messageFormatBlobPropertiesTest(short blobVersion, BlobType blobTyp
     ByteBufferInputStream stream = new ByteBufferInputStream(ByteBuffer.wrap(data));
 
     MessageFormatInputStream messageFormatStream =
-        (blobVersion == MessageFormatRecord.Blob_Version_V1) ? new PutMessageFormatInputStream(key, prop,
+        (blobVersion == MessageFormatRecord.Blob_Version_V2) ? new PutMessageFormatInputStream(key, prop,
```

Comment:
All cases in MessageFormatInputStreamTest are just an assignment statement. do you think we need a switch case for it. I felt ternary is good enough. I can change it for BlobStoreHardDeleteTest which I felt as reasonable